interfaces/cython/setup.cfg.in

```diff
@@ -49,6 +49,7 @@ packages =
     cantera.test
     cantera.test.data
     cantera.examples
+py_modules =
```

Comment:
I don't think this line is required. At least, the same files are packaged in the wheel whether this is here or not.

Reply:
This was based on reading of the (conflicting) documentation about this new "discovery" feature of `setuptools`, where it says:
> Projects that currently don’t specify both packages and py_modules in their configuration and contain extra folders or Python files (not meant for distribution), might see these files being included in the wheel archive or even experience the build to fail.

Elsewhere, it says:
> Automatic discovery will only be enabled if you don’t provide any configuration for packages and py_modules. If at least one of them is explicitly set, automatic discovery will not take place.

So I really don't know. But I guess we can see if everything still works without it.

Reply:
I have setuptools 61.1.1, so I'm not sure.

Their documentation is _terrible_, a mishmash of stuff copied from the original distutils (and still duplicated in the official Python docs), random pages of (maybe?) best practices, and no complete API specification. But hey, it looks pretty!